Mathlib/Algebra/ContinuedFractions/Computation/CorrectnessTerminating.lean

```diff
@@ -132,8 +132,8 @@ theorem compExactValue_correctness_of_stream_eq_some :
     -- Nat.succ
     obtain ⟨ifp_n, nth_stream_eq, nth_fract_ne_zero, -⟩ :
       ∃ ifp_n, IntFractPair.stream v n = some ifp_n ∧
-        ifp_n.fr ≠ 0 ∧ IntFractPair.of ifp_n.fr⁻¹ = ifp_succ_n :=
-      IntFractPair.succ_nth_stream_eq_some_iff.1 succ_nth_stream_eq
+        ifp_n.fr ≠ 0 ∧ IntFractPair.of ifp_n.fr⁻¹ = ifp_succ_n
+    exact IntFractPair.succ_nth_stream_eq_some_iff.1 succ_nth_stream_eq
```

Comment:
This seems to be reverting to flow-of-consciousness `obtain` proofs.
(Also in the rest of this file.)